docs/sources/reference/components/otelcol/otelcol.exporter.datadog.md

```diff
@@ -0,0 +1,324 @@
+`max_idle_conns_per_host` | `int`                      | Limits the number of idle HTTP connections the host can keep open.                                                 |         | no
+`max_conns_per_host`      | `int`                      | Limits the total (dialing,active, and idle) number of connections per host.                                        |         | no
+`idle_conn_timeout`       | `duration`                 | Time to wait before an idle connection closes itself.                                                              |     | no
+`disable_keep_alives`     | `bool`                     | Disable HTTP keep-alive.                                                                                           |     | no
+`insecure_skip_verify`         | `boolean`      | Ignores insecure server TLS certificates.                                                    |             | no
+
+
+### retry_on_failure block
+
+The `retry_on_failure` block configures how failed requests to Datadog are retried.
+
+{{< docs/shared lookup="reference/components/otelcol-retry-block.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+### queue block
+
+The `queue` block configures an in-memory buffer of batches before data is sent to the HTTP server.
+
+{{< docs/shared lookup="reference/components/otelcol-queue-block.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+### debug_metrics block
+
+{{< docs/shared lookup="reference/components/otelcol-debug-metrics-block.md" source="alloy" version="<ALLOY_VERSION>" >}}
+
+## Exported fields
+
+The following fields are exported and can be referenced by other components:
+
+Name    | Type               | Description
+--------|--------------------|-----------------------------------------------------------------
+`input` | `otelcol.Consumer` | A value other components can use to send telemetry data to.
+
+`input` accepts `otelcol.Consumer` data for any telemetry signal (metrics, logs, or traces).
+
+## Component health
+
+`otelcol.exporter.datadog` is only reported as unhealthy if given an invalid
+configuration.
+
+## Debug information
+
+`otelcol.exporter.datadog` does not expose any component-specific debug
+information.
+
+## Example
+
+This example forwards Prometheus metrics from {{< param "PRODUCT_NAME" >}} through a receiver for conversion to Open Telemetry format before finally sending them to Datadog.
+If you are using the US Datadog APIs, the `api` field is required for the exporter to function.
+
+```alloy
+prometheus.exporter.self "default" {
+}
+
+prometheus.scrape "metamonitoring" {
+  targets    = prometheus.exporter.self.default.targets
+  forward_to = [otelcol.receiver.prometheus.default.receiver]
+}
+
+otelcol.receiver.prometheus "default" {
+  output {
+    metrics = [otelcol.exporter.datadog.default.input]
+  }
+}
```

Comment:
for the example I would suggest to do a full otel pipeline instead of converting from Prometheus to Otel